conbench/app/_plots.py

```diff
@@ -286,7 +313,7 @@ def _inspect_for_multisample(items) -> tuple[bool, Optional[int]]:
 
     # Get number of samples for each benchmark.
     try:
-        samplecounts = [len(i["data"]) for i in items]
+        samplecounts = [len(i.data) for i in items]
```

Comment:
The major benefit of this patch is not easily seen by looking at the github diff view. It's really things like this:

![image](https://user-images.githubusercontent.com/265630/223410577-a60227fb-2a56-4ec4-ad93-875327ead3fc.png)

The IDE now knows what `i.data` is. So relieving.